extensions/wikia/Email/templates/avatarLayout.mustache

```diff
@@ -1,6 +1,6 @@
 <div class="greeting">
 	<p>{{salutation}}</p>
-	<p>{{{summary}}}</p>
+	<p><strong>{{{summary}}}</strong></p>
```

Comment:
I basically did a soft merge of the addition @mklucsarits already added in another email ticket.  Maybe work it out with him and I'll update to whatever you guys decide?